packages/runtime/src/controls/rich-text.ts

```diff
@@ -0,0 +1,129 @@
+import { IndexSignatureHack } from '../prop-controllers/descriptors'
+import type * as Slate from 'slate'
+import { BuilderEditMode } from '../utils/constants'
+import { PropController } from '../prop-controllers/instances'
+import { OnChangeParam, Editor } from 'slate-react'
+import { BoxModel } from '../box-model'
+
+export type RichTextControlData = IndexSignatureHack<Slate.ValueJSON>
+
+export const RichTextControlType = 'makeswift::controls::rich-text'
+
+type RichTextControlConfig = {}
```

Comment:
Something to be careful with is that in TypeScript `{}` is almost the same as `any`. If you want to model an empty object, you should use `Record<string, never>` instead. But really, we should just not have any arguments if there aren't any yet.

Reply:
I decided to work that muscle and remove this since there are no props yet